vue-components/src/components/Lookup.vue

```diff
@@ -26,6 +26,9 @@
 			@update:searchInput="$emit('update:searchInput', $event)"
 			@input="$emit('input', $event)"
 			@scroll="(firstIndex, lastIndex) => $emit('scroll', firstIndex, lastIndex)"
+			aria-autocomplete="list"
+			aria-haspopup="menu"
```

Comment:
This should maybe be `listbox` instead.

Also, I think we can add this in `LookupInput` component itself.

Mainly going by this example: https://www.w3.org/TR/2019/NOTE-wai-aria-practices-1.1-20190814/examples/combobox/aria1.1pattern/listbox-combo.html#ex1_label

Reply:
If it works this simple, then let's go with this. The simpler, the better. Though we might still need it later, but we can figure that out in a second iteration.